srcs/Channel.cpp

```diff
@@ -54,7 +55,20 @@ void	Channel::addOperator(std::string operatorName)
 void	Channel::removeOperator(std::string operatorName)
 {
 	std::vector<std::string>::iterator it = std::find(_operators.begin(), _operators.end(), operatorName);
-	_operators.erase(it);
+	if(it != _operators.end())
+	{
+		_operators.erase(it);
+		if(!_operators.empty())
+			std::cout << _operators.back() << "is the new operator of the channeeeel" << std::endl;
```

Comment:
funny message but let's change channeeeel to channel

Reply:
Done it!